src/platform/windows.rs

```diff
@@ -82,7 +82,7 @@ pub trait WindowExtWindows {
     fn set_taskbar_icon(&self, taskbar_icon: Option<Icon>);
 
     /// Whether the system theme is currently Windows 10's "Dark Mode".
-    fn is_dark_mode(&self) -> bool;
+    fn theme(&self) -> Theme;
```

Comment:
This seems like a breaking change, so it should be listed in the changelog

Reply:
Changed the comment